crates/datafusion/src/logical_plan/expr.rs

```diff
@@ -1215,13 +1215,23 @@ pub trait Literal {
 
 impl Literal for &str {
     fn lit(&self) -> Expr {
-        Expr::Literal(ScalarValue::LargeUtf8(Some((*self).to_owned())))
+        //FIXME debug_assert!(self.len()<128);
```

Comment:
This is totally wrong. We can not expand the literal string. Because this will make the semantics of string functions changed.